server.go

```diff
@@ -883,6 +901,9 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 	method := sm[pos+1:]
 	srv, ok := s.m[service]
 	if !ok {
+		if handled := s.processUnknownStreamHandler(t, stream, trInfo); handled {
```

Comment:
Function `processUnknownStreamHandler` is not so necessary now.
I think we could just check `s.opts.unknownStreamDesc` and call `s.processStreamingRPC` here directly.